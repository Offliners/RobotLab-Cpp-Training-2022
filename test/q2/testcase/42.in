273
CrYsTAL
AgAI7
destrUCTiVe
RIOt
TOP
inFluential
pedestriAN
OBSERVATION
MUScLE
CHOcOLATE
appreciaTion
campaign
INch
DiSPUTE
SEcURE
Sock
cENTrAl
REsIsTANCE
KeTChUP
regiment
SEvEnteeN
loft
DWARF
luXUry
I
explore
03li58rAt6ly
distribute
00etTo
SpOiL
CongraTuLatIoN
DIAPER
GAmBLE
exPeCtatIoN
inTEnsity
CARPENTER
halt
ACCUSaTIoN
FEET
DRaG
diSh
CONSENT
GENtleMAN
NurSe
PolitiCs
Expire
heEl
OwNErshIP
buRSt
suIt
PART
HyPoCriSy
COLLAPSE
heAVIeR
PeSTICIDE
ChILlY
engINe
LegACy
ARRAY
clOSET
cuBIc
PARaDISE
grAve
lobby
advise
oven
ADeQUaTELY
HuMIlIAte
tRIBUNE
SEND
CAUTIOUSLY
aMAzE
equAlItY
balaNCE
EXIstEnce
pineaPple
mErCy
fund
refUsAL
CASINO
PATrIOTIC
ErASE
tIgHt
c861
BUsH
EDUCATE
SERUm
bAn
BAsEMENT
Vocal
IndoorS
roYalTY
518P1
FLoUR
vacUuM
rEPLACEABle
ELeCTROnic
8riti1
independeNcE
Broad
lAy
touchSTONe
RIDICULOUS
cITIZensHIP
pOssible
HARNESS
RAnChER
tIme
instAllation
bRICK
cOnSPIRaCY
EXTRACURRICULAR
share
PLAYGRouNd
siGhTSeEing
slum
LaTE
34t756
volleyball
WEB
UnREST
impORTANT
Battle
mind
infinite
pUT
HUMANITY
diSSAtISFY
s7666
DISpOSAbLe
CHECkBOOK
MAJEStY
clothes
nutrition
14L
dRape
MOp
Sin
9herapi1t
ARtIcUlATE
DECEASE
CUSHiON
raInFALL
BLANd
793o1
sUrge
BACK
PRIVACY
InTroduCE
imperial
humbly
BIzARRe
lightweight
REGULaR
SUppLy
REMain
GrOup
welfare
processing
ENTAIL
FuLFILl
violate
SuCk
FRESHmAN
INVENT
HALl
LeGaL
perplex
ginger
LATEST
WHAT
NeEDy
ThREAtEN
PoCkET
PopulAtioN
ClOwn
CoW
VISIBLE
DIStURB
cOArse
rOW
vOlunteer
GRAPEFRUIt
card
sEason
CONtinuE
OutcOME
UTTeRlY
IRONY
LeNs
RELAtivE
RobiN
PUrSuE
recolleCt
range
Pat
CrEEP
pig
CoMe
stALE
BArK
SAiL
harm
sIX
signifiCant
680CTUAl
MatErial
cOckrOAch
SA55
COUNTABLE
dread
SCRIPT
WHiLe
rEveRse
ADVANTAGE
DIgNify
bug
HOSTILE
mANIPULATE
casual
INFORM
sLIGhT
UraNIuM
aTTIc
HELMET
house
RUsT
FORWARD
OVERCOAT
sticky
INTENsE
OWn
ingenuity
humiliate
painfully
fAt
clockwork
SouvENIr
CoLLEGe
casTE
pOultrY
again
spoon
BracE
voTEr
inheRit
PRize
CO2BOY
GoLd
restroom
TeLleR
PoLLUTiON
OFFSPrING
timid
MEMORIZE
PROPOSE
incReDible
CONSTABLe
rubbeR
criTicIsm
socIaLlY
pUzzle
creatIvity
classiCal
CRADLE
THRIVE
DrAGOn
DEFinIte
PreFer
bREaTH
GUARANTEE
cardinaL
alongside
