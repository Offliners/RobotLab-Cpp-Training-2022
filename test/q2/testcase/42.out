adj./n. 水晶的
Unknown
adj. 毀滅性的
n. 暴動
adj./nJv. 頂端的；頂部
adj. 有影響的
n. 行人
n. 觀察
n. 肌肉
n. 巧克力
n. 欣賞；感謝
n./v. 競選活動；戰役
n./v. 英吋
v./n. 爭論
adj. 安全的
n. 短襪
adj. 中央的；核心的
n. 反抗
n. 調味蕃茄醬
n. (軍)團
adj./n. 十七
n. 閣樓
n./adj. 侏儒；矮小的
n. 奢侈
Unknown
v. 探索
Unknown
v. 分配
Unknown
v. 破壞
n. 恭喜
n. 菱形花紋(布)；尿布
v. 賭博；投機
n. 期望
n. 強烈；強度
n. 木匠
v./n. 停止；暫停
n. 控訴
n. [pl. ]腳
v. 拖曳
n. 碟；菜餚
n./v. 同意
n. 紳士
n. 護士
n. 政治活動
v. 期滿
n. 腳後跟
n. 所有權
v./n. 爆發；爆炸
n. 一套衣服
n. 一部分
n. 虛偽
v./n. 崩塌
adj. 重的
n. 殺蟲劑
adj.  冷颼颼的
n. 發動機；引擎
n. 遺產
n. 一長列
n. 壁櫥；衣櫥
adj. 立方的
n. 天堂
adj. 重大的；嚴肅的
n. 門廊；休息室
v. 勸告；忠告；通知
n. 烤箱
adv. 充分地
v. 屈辱
n. 護民官
v. 送
adv. 小心地
v. (n)使驚愕
n. 相等；平等；均等
n./v. 平衡
n. 存在
n. 鳳梨
n. 慈悲
n./v. 資金
n. 拒絕
n. 賭場
adj. 愛國的
v. 擦掉；抹去
adj./adj. 緊的；緊緊地
Unknown
n. 灌木叢
v. 教育；教導
n. 血清
v. 禁止
n. 地下室
adj. 發音的
adv. 在室內
n. 皇室成員
Unknown
n. 麵粉
n. 真空；空虛(感)
adj. 可取代的
adj. 電子的
Unknown
n. 獨立
adj. 寬廣的；概括的
v. 放；安排；下蛋
n. 試金石；(檢驗)標準
adj. 可笑的
n. 公民身分(權利)
adj. 可能的
n. 馬具
n. 農場主人
n./v. 時間
n. 安置
n. 磚
n. 陰謀
adj. 課外的
n./v. 分擔；分配
n. 操場
n. 觀光
n. 貧民窟
adj./adv. 遲的；晚的
Unknown
n. 排球
n. 網
n. 動亂
adj. 重要的
n./v. 戰役
n./v. 頭腦；注意；留意
adj. 無窮的
v. 安置
n. 人性；人文學科
v. 使不快
Unknown
adj. 可丟棄的
n. 支票簿
n. 威嚴；王權
n. 衣服
n. 營養；營養學
Unknown
v. 覆蓋
n./v. 拖把
n./v. 罪惡
Unknown
v. 清楚的表達
n. 死亡
n. 墊子
n. 降雨
adj. 溫和的
Unknown
v. 洶湧向前
adj./adv./n./v.  後面的；背部；贊助
n. 獨處(狀況)
v. 介紹；提出
adj. 帝國的
adv. 謙虛地
adj. 古怪的
n./adj. 平均重量以下；不重要的人
adj. 整齊的；規律的
n./v. 供應
v. 殘留
n./v. 群；團體；聚集
n. 福利
n. 過程
v. 使成為必要
v. (n)實現
v. 違背；侵犯
v. 吸；吮
n. 新鮮人
v. 發明；創造
n. 大廳；走廊
adj. 法律上的；合法的
v. 使困惑
n. 生薑
adj. 最近；最遲
pron./conj. 什麼
adj. 非常貧窮的
v. 恐嚇
n. 口袋
n. 人口
n. 小丑
n. 母牛
adj. 看得見的
v. 妨礙；打擾
adj. 粗糙的
n./v. 爭吵
n./v. 志願者；自願
n. 葡萄柚
n. 卡片；紙牌遊戲
n. 季節
v. 持續
n. 結果
adj. 完全地
n. 反語
n. 鏡片；眼球的水晶體
adj. 比較上的
n. 知更鳥
v. 追捕
v. 想起
n. 區域
n. 拍
v. 躡手躡腳地走
n. 豬
v. 來到；甦醒
adj. 不新鮮的
n./v. 吠叫；咆哮
n./v. 船；坐船旅行
n./v. 損害；傷害
adj./n. 六
adj. 重要的
Unknown
n. 材料
n. 蟑蚣卜；油蟲
Unknown
adj. 可數的
n./v. 害怕
n. 原稿
conj/n. 當…的時候
adj./n./v. 相反的；反轉
n. 有利條件；優勢
v. 使有尊嚴
n. 蟲子
adj. 敵人的
v. 操縱
adj. 偶然的；碰巧的
v. 通知
adj. 輕微的
n. 鈾
n. 閣樓
n. 頭盔；安全帽
n./v. 住宅
n./v. 銹；生銹
adj./v. 向前；今後
n. 外套大衣
adj. 黏的
adj. 強烈的
adj./v. 自己的
n. 靈巧
v. 屈辱
ad v. 痛苦地
adj./n. 肥胖的
n. 發條裝置
n. 紀念品
n. 大學；學院
n. 種姓(制度)
n. 家禽(肉)
adv. 再一次
n./v. 匙；用匙舀
n. 支撐物
n. 選民
v. 繼承
n./v. 獎品；珍視
Unknown
n./adj. 黃金(色)
n. 廁所
n. 講故事者；銀行出納員
n. 污染
n. 後代；後裔
adj. 膽小的
v. 背誦；記憶
v. 提議
adj. 驚奇的
n. 警察
n. 橡膠
n. 評論
adj. 善於交際地
n./v. 難題；困惑
n. 創造力
adj. 極好的；古典的
n./v. 搖籃；撫育
v. 興旺
n. 龍
adj. 明確的；肯定的
v. 寧願選擇
n. 呼吸
n./v. 保證；擔保品
n. 紅衣主教
prep./adv. 靠著…旁邊
